site/content/3.10/develop/http-api/administration.md

```diff
@@ -796,6 +803,12 @@ paths:
                       - `read-only`: The license is expired over 2 weeks. The instance is now
                         restricted to read-only mode.
                     type: string
+                    example: good
+                  upgrading:
+                    description: |
+                      Whether a license upgrade is in progress.
```

Comment:
This is just a wild guess, please clarify!

Reply:
It seems to indicate whether the server is in the database upgrade mode, i.e. started with `--database.auto-upgrade true` or `--database.check-version true`